src/mcp/state.py

```
@@ -30,164 +76,425 @@
-        """Export the configuration state to a YAML file.
+        try:
+            with tempfile.NamedTemporaryFile(
+                mode="w", suffix=".idf", delete=False
+            ) as tf:
+                tf.write(self.seed_idf_text)
+                loaded = IDF.load(Path(tf.name))
+                Path(tf.name).unlink()
+                new_idf = IDF.from_dict(loaded.to_dict())
+            self._set_idf_private(new_idf)
+            return True
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)